src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -110,14 +124,15 @@
       "commercetools-project-sync.runnerName.ProductSync.timestampGenerator";
 
   @BeforeEach
-  void setup() {
+  void setup() throws ExecutionException, InterruptedException {
     syncerTestLogger.clearAll();
     cliRunnerTestLogger.clearAll();
     cleanUpProjects(createClient(CTP_SOURCE_CLIENT_CONFIG), createClient(CTP_TARGET_CLIENT_CONFIG));
     setupSourceProjectData(createClient(CTP_SOURCE_CLIENT_CONFIG));
   }
 
-  static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClient) {
+  static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClient)
+      throws ExecutionException, InterruptedException {
```

Comment:
Remove these throws and use `join` instead of `get`.

Reply:
These changes are not related to this PR.

Reply:
Maybe not anymore in the new PR, but here it was in the PR and if you don't fix them now, they will never be fixed again 😃

Reply:
please also suggest me, where to log the time.